src/xdocs/config_filters.xml

```diff
@@ -2232,6 +2232,275 @@ public class UserService {
+        <source>
+int a = 42; // ok
+int longVarName = 2; // LineLength violation
+        </source>
+        <p>
+          To configure the filter to suppress audit events for the current and next 2 lines:
+        </p>
+        <source>
+&lt;module name=&quot;Checker&quot;&gt;
+  &lt;module name=&quot;SuppressWithNearbyTextFilter&quot;&gt;
+    &lt;property name=&quot;lineRange&quot; value=&quot;2&quot;/&gt;
+  &lt;/module&gt;
+  &lt;module name=&quot;UniqueProperties&quot;/&gt;
+&lt;/module&gt;
+        </source>
+        <p>
+          Example:
+        </p>
+        <source>
+key.one=41 # SUPPRESS CHECKSTYLE the reason for this is
+key.one=42 # ok
+key.one=43 # ok
+key.one=44 # violation
+        </source>
+        <p>
+          To configure the filter to suppress audit events for the current and previous line:
+        </p>
+        <source>
+&lt;module name=&quot;Checker&quot;&gt;
+  &lt;module name=&quot;SuppressWithNearbyTextFilter&quot;&gt;
+    &lt;property name=&quot;lineRange&quot; value=&quot;-1&quot;/&gt;
+  &lt;/module&gt;
+  &lt;module name=&quot;UniqueProperties&quot;/&gt;
+&lt;/module&gt;
+        </source>
+        <p>
+          Example:
+        </p>
+        <source>
+key.one=41 # violation
+key.one=42 # ok
+key.one=43 # SUPPRESS CHECKSTYLE the reason for this is
+key.one=44 # violation
+        </source>
+        <p>
+          To configure the filter with a more compact <code>nearbyTextPattern</code>
+          to accept variable <code>checkPattern</code>:
+        </p>
+        <source>
+&lt;module name=&quot;Checker&quot;&gt;
+  &lt;module name=&quot;SuppressWithNearbyTextFilter&quot;&gt;
+    &lt;property name=&quot;nearbyTextPattern&quot;
+      value=&quot;-@cs\[(\w+)\] (\w+)&quot;/&gt;
+    &lt;property name=&quot;checkPattern&quot; value=&quot;$1&quot;/&gt;
+  &lt;/module&gt;
+  &lt;module name=&quot;TreeWalker&quot;&gt;
+    &lt;module name=&quot;MagicNumber&quot;/&gt;
+  &lt;/module&gt;
+&lt;/module&gt;
+        </source>
+        <p>
+          Example:
+        </p>
+        <source>
+int a = 42; // -@cs[MagicNumber] We do not consider this number as magic for some reason.
+int b = 43; // violation
+        </source>
+        <p>
+          To configure the filter to accept variable <code>checkPattern</code>
+          and <code>lineRange</code>:
+        </p>
+        <source>
+&lt;module name=&quot;Checker&quot;&gt;
+  &lt;module name=&quot;SuppressWithNearbyTextFilter&quot;&gt;
+    &lt;property name=&quot;nearbyTextPattern&quot;
+      value=&quot;@cs-: (\w+) - ([+-]\d+) Lines&quot;/&gt;
+    &lt;property name=&quot;checkPattern&quot; value=&quot;$1&quot;/&gt;
+    &lt;property name=&quot;lineRange&quot; value=&quot;$2&quot;/&gt;
+  &lt;/module&gt;
+  &lt;module name=&quot;TreeWalker&quot;&gt;
+    &lt;module name=&quot;MagicNumber&quot;/&gt;
+  &lt;/module&gt;
+&lt;/module&gt;
+        </source>
+        <p>
+          Example:
+        </p>
+        <source>
+int a = 42; // @cs-: MagicNumber - +3 Lines
```

Comment:
Remove `-` use `for`. Lowercase `Lines`.

Reply:
Done.